spec/components/schemas/content/carousel.ts

```diff
@@ -6,63 +6,64 @@ import { createComponentRef } from '../../../../utils/ref';
 
 const carousel: SchemaObject = {
   type: 'object',
-  allOf: [{
+  allOf: [
+    {
     $ref: baseRef,
```

Comment:
O NPM Lint ainda está impedindo a build. Está apontando pra esse arquivo. Log abaixo:

/home/travis/build/zenvia/zenvia-openapi-spec/spec/components/schemas/content/carousel.ts:11:1
ERROR: 11:1  ter-indent              Expected indentation of 6 spaces but found 4.
ERROR: 12:7  no-trailing-whitespace  trailing whitespace
ERROR: 63:8  trailing-comma          Missing trailing comma

Reply:
Erro corrigido com ajuda do Victor.